### PR TITLE
Feature/8065 click request message open conversation in slack

### PR DESCRIPTION
**Pull request by melsawy**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -247,6 +247,11 @@ const process = function(event, callback, teamConfig) {
           const value2 = { team_slug: teamSlug, annotation_id: resp.annotation.id, mode: 'bot' };
           const message = { text: t('project_set') + ': ' + projectUrl, response_type: 'in_channel', token: ACCESS_TOKEN, channel: event.channel };
 
+          // Store SmoochUserSlackChannelUrl
+          setSmoochUserSlackChannelUrl(event, { teamId: teamConfig.teamId, id: resp.annotation.dbid }, config.checkApi.apiKey, callback, function(resp) {
+            console.log('Added smooch user slack channel url to smooch user annotation');
```
Comment (caiosba) [inline index.js]: Please add some information to the log that we can identify the object... for example, the annotation id

File: index.js [hunk 2]
```diff
@@ -607,6 +612,24 @@ const markTranslationAsError = function(event, data, token, callback, done) {
   executeMutation(mutationQuery, vars, sendErrorMessage, done, token, callback, event, data);
 };
 
+const setSmoochUserSlackChannelUrl = function(event, data, token, callback, done) {
+  const slackChannelUrl = 'https://app.slack.com/client/' + data.teamId + '/' + event.channel
```
Comment (caiosba) [inline index.js]: Please add a `;` to the end of the line

File: index.js (hunk 1 above)
Comment (melsawy) [reply index.js]: Done

File: index.js (hunk 2 above)
Comment (melsawy) [reply index.js]: Done

File: index.js [hunk 3]
```diff
@@ -266,6 +266,11 @@ const process = function(event, callback, teamConfig) {
           const value2 = { team_slug: teamSlug, annotation_id: resp.annotation.id, mode: 'bot' };
           const message = { text: t('project_set') + ': ' + projectUrl, response_type: 'in_channel', token: ACCESS_TOKEN, channel: event.channel };
 
+          // Store SmoochUserSlackChannelUrl
+          setSmoochUserSlackChannelUrl(event, { teamId: teamConfig.teamId, id: resp.annotation.id, dbid: resp.annotation.dbid }, config.checkApi.apiKey, callback, function(resp) {
+            console.log('Added smooch user slack channel url to smooch user annotation' + data.dbid);
```
Comment (caiosba) [inline index.js]: Please add a space after "annotation"

**Pull request merged**